src/main/java/seedu/address/logic/commands/SortCommand.java

```diff
@@ -23,8 +23,6 @@ public CommandResult execute(Model model) throws CommandException {
         if (noAppointment == -1) {
             throw new CommandException(NO_APPOINTMENT_FOUND);
         }
-        model.updateFilteredPersonList(Model.PREDICATE_SHOW_ALL_PERSONS);
-
```

Comment:
Umm just asking what does this do exactly? Coz I did not include this but the features still work as intended.

Reply:
Are you asking about the line 26?

Reply:
Yep

Reply:
Updated. I think it was there from the earlier implementation. I had removed it I thought but must have missed it.

Reply:
Oh sorry, what I meant is I did not remove this line but the features work as intended. It's also used in most Command classes

Reply:
I know. But it did not require in this one cause we calling the sortedFilteredPersons() so I removed it

Reply:
In other Command classes we need to update the list based on one particular predicate so we called it

Reply:
Ohhh ok thanks for explaining!